src/frontend/src/components/TimeSlot.tsx

```diff
@@ -22,18 +24,20 @@ const TimeSlot: React.FC<TimeSlotProps> = ({
   onMouseDown,
   onMouseEnter,
   onMouseUp,
-  onMouseOver
+  onMouseOver,
+  onClick,
+  selected = false
 }) => {
   return (
     <Box
       sx={{
         height: small ? '25px' : '4.7vh',
         width: small ? '81px' : '12.2%',
-        backgroundColor,
+        backgroundColor: backgroundColor,
         cursor: onMouseEnter ? 'pointer' : undefined,
         borderStyle: 'solid',
-        borderColor: 'gray',
-        borderWidth: '0.1px',
+        borderColor: selected ? '#ffff8c' : 'gray',
```

Comment:
the highlighted color should probably be red or NER red instead of yellow

Reply:
@Aaryan1203 suggested yellow, I'm assuming because once the calendar fills with people's availabilities, the slots will be different shades of red, and the yellow border color would make the selected slot more visible than red

Reply:
ohh ok gotcha, yellow's cool then